src/plugin/class-post-type-manager.php

```diff
@@ -24,6 +24,10 @@ public function __construct() {
 		add_action( 'init', array( $this, 'register_post_types' ) );
 	}
 
+	private function is_local_env(): bool {
```

Comment:
```suggestion
	private function should_show_ui(): bool {
```

The advantage would be that if we want more elaborate logic in the future for whether the ui should be shown, it can be added in this function, without requiring changing other places of the code.